met-web/src/components/common/MetTabs/MetTabs.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import Box from '@mui/material/Box';
+import TabContext from '@mui/lab/TabContext';
+import { MetTab, MetTabList } from './StyledTabComponents';
+
+interface MetTabProps {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    tabs: any | undefined;
+    updateTabName: (arg: string) => void;
+}
+
+const MetTabs: React.FC<MetTabProps> = ({ tabs, updateTabName }) => {
+    const [value, setValue] = React.useState(`${Object.keys(tabs[0])}`);
+    useEffect(() => {
+        updateTabName(value);
+    });
+
+    return (
+        <Box sx={{ width: '100%', typography: 'body1' }}>
+            <TabContext value={value}>
```

Comment:
Also the Tab panel is expected to be under TabContext. In this case only the tabs exist without the panel? It doesn't follow MUI's documentation 
![image](https://user-images.githubusercontent.com/91914654/208714735-fe4e0106-c307-48c9-a520-8914573ec1f7.png)

Reply:
Changed the logic to have separate code for survey block tabs